File: public/nootan-20160520092127/PYTHON/sherlock.py
#!/usr/bin/python
# Name : Sherlock
# input:
# n : size
# m : size
# map : ratio
elements = input().split(' ')
n = int(elements[0]);
m = int(elements[1]);
for _ in xrange(n):
	map_elements = input().split(' ')
	map = (e for e in map_elements)


# write your code here
# store your results in `row`,`column`

# output
print(str(row)+" "+str(column));

File: public/nootan-20160520092127/C/sherlock.c
// Name : Sherlock
#include <stdio.h>
int main() {
	// input:
	// n : size
	// m : size
	// map : ratio
	int n, m, row, column, idx, jdx;
	char map[100][100], tmp[100];
	scanf("%d %d", &n, &m);
	for(idx = 0; idx< n; idx++) {
		scanf("%s", &tmp[0]);
		for(jdx = 0;jdx<m;jdx++) {
			map[idx][jdx] = tmp[jdx];
		}
	}


	// write your code here
	// store your results in `row`,`column`

	// output
	printf("%d %d", row, column);
	return 0;
}